hooks/test

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
This file is pulled almost exactly from `circulation/docker-test`. I'll note some changes in additional commends.